modules/luci-base/htdocs/luci-static/resources/cbi.js

```diff
@@ -630,8 +630,8 @@ String.prototype.format = function()
 						}
 
 						subst = (td > 0)
-							? String.format('%dd %dh %dm %ds', td, th, tm, ts)
-							: String.format('%dh %dm %ds', th, tm, ts);
+							? String.format('%d:%d:%d:%d', td, th, tm, ts)
```

Comment:
I am not that fluent in JS, but this looks to be printf-style formatting. If it is, "%d:%d" results in "2:2" for 2 and 2 as input. For numbers representing hours, minutes or seconds, you would always expect two digits: potentially leading zeroes: "02:02".